src/main/java/appeng/client/me/ItemRepo.java

```diff
@@ -51,6 +57,13 @@ public class ItemRepo {
     private String innerSearch = "";
     private String NEIWord = null;
     private boolean hasPower;
+    private static final ListMultimap<Class<? extends IAEStack<?>>, BiPredicate<IAEStack<?>, Enum<TypeFilter>>> filters = ArrayListMultimap
+            .create(4, 8);
+
+    public static <StackType extends IAEStack<StackType>> void registerTypeHandler(
+            BiPredicate<IAEStack<?>, Enum<TypeFilter>> filter, Class<StackType> stackTypeClass) {
```

Comment:
Why `Enum<TypeFilter>` and not just `TypeFilter`?

Reply:
fixed